README.md

```diff
@@ -396,14 +397,59 @@ authClient.signIn({
 
 ### `signOut()`
```

Comment:
I think we need to generally improve the documentation of this method.  The primary tradeoff here is "Do I want to remove the Okta session during logout, or not?" and I don't think we're doing a good job of laying out that tradeoff.  That should introduce that concept before getting into the options and underlying behaviors.